main.tex

```diff
@@ -42,6 +42,12 @@
 \definecolor{color_29791}{rgb}{0,0,0}OWNERS CORPORATION RULES Owners Corporation 1 on Plan of Subdivision PS834362N Brunswick Yard 8 Ballarat Street BRUNSWICK VIC 3056
 \newpage
 
+\section[intro]{Introduction}
+As legislated in the \href{https://www.consumer.vic.gov.au/housing/owners-corporations/rules/model-rules#:~:text=The%20Model%20rules%20for%20an%20owners%20corporation%20%28Word%2C,the%20model%20rules%2C%20then%20the%20model%20rule%20applies.}{Owners Corporations Regulations}, if these following rules fail to discuss something prescribed as a model rule, then the model rule applies.
+
```

Comment:
This note about the legislation should've really been present all along. It is not obvious to readers of the current rules that there are also rules being imposed by the Victorian government.